CodeEdit/CodeEditApp.swift

```diff
@@ -6,11 +6,15 @@
 //
 
 import SwiftUI
+import WindowManagement
 
 @main
 struct CodeEditApp: App {
     @NSApplicationDelegateAdaptor var appdelegate: AppDelegate
     @ObservedObject var settings = Settings.shared
+
+    @Environment(\.openWindow) var openWindow
```

Comment:
This reads like an action. Does this refer to the current window that is open? Or the current window that is active?

Reply:
This is an [action](https://developer.apple.com/documentation/swiftui/environmentvalues/openwindow) to open a new window